Artifacts/linux-eris-blockchain/docker.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+# -----------------------------------------------------------------------------
+# PURPOSE
+
+# This script will setup docker.
+
+# **NOTE** -- This script is used by Eris to provision test box backends which
+# we need to be on a specific version of docker. It will likely be unuseful to
+# you.
+
+# If you are looking for a quick and easy way to set up eris on a cloud machin
+# please see https://raw.githubusercontent.com/Azure/azure-devtestlab/master/Artifacts/linux-eris-blockchain/setup.sh
+
+# If $DOCKER_VERSION is set then the host will use that.
+
+# -----------------------------------------------------------------------------
+# LICENSE
+
+# The MIT License (MIT)
+# Copyright (c) 2016-Present Eris Industries, Ltd.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+# -----------------------------------------------------------------------------
+# REQUIREMENTS
+
+# Ubuntu
+
+# -----------------------------------------------------------------------------
+# USAGE
+
+# docker.sh
+
+# -----------------------------------------------------------------------------
+# Set defaults
+
+default_docker="1.9.1"
+
+# -----------------------------------------------------------------------------
+# Check Ubuntu
+
+read -p "This script only works on Ubuntu (and does no checking). It may work on some debians. Do you wish to proceed? (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
```

Comment:
These scripts will not be run interactively. You need to pipe in parameters from the artifact if you need to modify script behavior, or make the actions understood by the user at Artifact selection/configuration time.